src/ui/atoms/button/Button.tsx

```diff
@@ -27,24 +37,29 @@ export type TButtonProps = Readonly<{
  * Primary UI component for user interaction.
  */
 export const Button: React.FC<TButtonProps> = ({
-  primary = false,
+  variant = 'primary',
   size = 'medium',
+  backgroundColor = 'primary',
+  disabled = false,
   label,
   onClick,
   ...props
```

Comment:
Could be removed. (`props` has `{}` type as far as I see).

Reply:
Yes you are right, and I realize that this problem is present on the other atomic components and needs to be fixed.
I propose initially to delete this element and thus to partition the use of the button to the properties defined by the interface.
Subsequently, we can discuss whether we extend the type to that of the `HTMLButtonElement`

Reply:
Good to me!